pkg/provider/oneview/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"os"
+
+	log "github.com/Sirupsen/logrus"
```

Comment:
We're moving away from logrus.  The new logging system does call stacks, line numbers, filter by matches, and levels.  

Use 
```
import (
  logutil "github.com/docker/infrakit/pkg/log"
)

var log = logutil.New("module", "instance/oneview")
var debugV = logutil.V(200)  // 100-500 are for typical debug levels, > 500 for highly repetitive logs (e.g. from polling)
```
Then use `log.Info`, `log.Debug` as needed.  You can add a verbose level to the debug log like so:
```
log.Debug("this is log header/message", "field1", "value1", "field2", "value2", "V", debugV)
```
Then in the CLI:
```
infrakit .... --log 5 --log-debug-V 500 --log-debug-match module=instance/oneview
```
will log only DEBUG statements from your package (module=instance/oneview) and any thing with V < 500.

Reply:
Cool, i'll move over to that logging package and fix the vending.